test/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/GetMessagesControllerSpec.scala

```diff
@@ -83,7 +83,7 @@ class GetMessagesControllerSpec extends PlaySpec
       contentAsJson(result) mustBe MovementIdFormatError
     }
 
-    "return 200" in {
+    "return 200 when consignor or consignee are valid" in {
```

Comment:
Technically doesn't this just check that the consignor is valid? The consignee isn't part of the retrievals from auth. Should have a separate test for being valid because of consignee